src/styles/index.css

```diff
@@ -0,0 +1,243 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: rgb(188, 202, 183);
+}
+
+h2 {
+  color: blue;
+}
+
+nav {
+  display: flex;
```

Comment:
- [ ] You have used `display: flex` in 10 CSS elements. This falls under the category of bad practicing because you are repeating yourself a lot. I would suggest you create a class similar to `d-flex` and give the display: flex there and use it at all the elements that you need